HTML/HTML.sublime-syntax

```diff
@@ -41,6 +41,8 @@ variables:
       | text/livescript
     )
 
+  tag_name: '[[:alpha:]_][[:alnum:]:_.-]*'
```

Comment:
https://html.spec.whatwg.org/multipage/parsing.html#tag-name-state

It might be worthwhile to mimic the standard parsing algorithm, which allows almost anything after the first character of a tag name.

Reply:
Took that entity from the existing XML syntax. Are there differences?

Reply:
Yes. The HTML5 spec defines different HTML and XML parsing modes.

Reply:
But looks like HTML is more permissive with DOCTYPE names than XML.

https://html.spec.whatwg.org/multipage/parsing.html#doctype-name-state